Document/0x06b-Basic-Security-Testing.md

```diff
@@ -176,13 +163,60 @@ $ docker run -it -p 8000:8000 opensecurity/mobile-security-framework-mobsf:lates
 
 #### Accessing the Device Shell
 
-##### On-device Shell App
+One of the most common things you do when testing an app is accessing the device shell. In this section we'll see how to access the iOS shell both remotely from your host computer with/without a USB cable and locally from the device itself.
 
 ##### Remote Shell
 
-##### Connecting to a physical iOS device
+In contrast to Android where you can easily access the device shell using the adb tool, on iOS you only have the option to access the remote shell via SSH. This means also that your iOS device must be jailbroken in order to connect to its shell from your host computer. For this section we assume that you've properly jailbroken your device and have either Cydia (see screenshot above) or Sileo installed as introduced in "Getting Privileged Access". In the rest of the guide we will reference to Cydia, but the same packages should be available in Sileo.
```

Comment:
This means also -> This also means

introduced -> explained

Reply:
done